### ISSUE TITLE
Хранить статистику в SQLite3

### ISSUE THREAD

**Issue by zedxxx** [Allexin/TrackYourTime#40]
Сейчас статистика хранится в каком-то самописном бинарном формате, соответственно, анализ её сторонними средствами сильно затруднён. Было бы гораздо удобнее, если бы статистика лежала в SQLite3.

**Comment by f2404** [comment]
:+1:

**Comment by Allexin** [comment]
Я не умею работать с SQLite и вообще внешними БД.
Конечно, я могу написать примитивное приложение, а по мануалам и что-то сложное. Но считаю, что в такой ситуации надо очень хорошее знание принципов работы с БД, чтобы все правильно сделать.
Поэтому перевод на БД(любую) делать в ближайшее время не буду. Когда не будет других задач, в экспериментальном режиме - возможно. Но точно не сейчас.
Если вдруг кто-то захочет сделать это самостоятельно, то pull request приму, только если это будет максимально независимый модуль с возможностью переключится на него в настройках.

**Comment by zedxxx** [comment]
Мысли вслух:

SQLite - встраиваемая БД и работать с ней весьма просто. Единственная сложность которая может возникнуть - придумать оптимальную схему БД, но с этим, я думаю, сообщество могло бы помочь. 

По поводу независимого модуля - полностью поддерживаю. Более того, текущую реализацию так же нужно сильно рефакторить (в `cDataManager` та ещё каша).

**Comment by Allexin** [comment]
Да, убер класс получился. А станет еще хуже в следующей версии, потому что добавится работа с подклассами приложений.
